cookbook/entities/acl.rst

```diff
@@ -0,0 +1,246 @@
+How to Protect Resources Using ACLs
+===================================
+
+Using ACLs you can granularly grant access to your entities. Doing so requires three steps:
+
+#. :ref:`Activate ACL checks for your entities <coobook-entities-acl-enable>`.
+
+#. :ref:`Create access control lists for all available actions <coobook-entities-acl-create>`.
+
+#. :ref:`Add access checks<coobook-entities-acl-check>` to where your entities are displayed or
+   manipulated.
+
+.. _coobook-entities-acl-enable:
+
+Activating ACL Checks on your Entities
+--------------------------------------
+
+In order to have your entity available in the admin UI to be able to assign permissions to your
+users you have to enable ACLs for these entities using the ``@Config`` annotation:
+
+.. code-block:: php
+    :linenos:
+
+    // src/AppBundle/Entity/Task.php
+    namespace AppBundle\Entitiy;
+
+    use Oro\Bundle\EntityConfigBundle\Metadata\Annotation\Config;
+
+    /**
+     * @Config(
+     *     defaultValues={
+     *         "security"={
+     *             "type"="acl",
+     *             "group_name"=""
+     *         }
+     *     }
+     * )
+     */
+    class Task
+    {
+        // ...
+    }
+
+After you have done this and have cleared the cache you can toggle all kinds of permission checks
+(``CREATE``, ``EDIT``, ``DELETE``, ``VIEW``, and ``ASSIGN`` in the user role management interface.
+
+.. tip::
+
+    You can use the optional group attributes to create visual groups in the UI that can help the
+    user find your entity.
+
+.. _coobook-entities-acl-create:
+
+Creating Access Control Lists
+-----------------------------
+
+You have to options to define your custom access control lists:
+
+.. _cookbook-entity-acl-controller:
+
+#. In your controller class, you can use the ``@Acl`` annotation:
+
+   .. code-block:: php
+       :linenos:
+
+       // src/AppBunde/Controller/TaskController.php
+       namespace AppBundle\Controller;
+
+       use AppBunde\Entity\Task;
+       use Oro\Bundle\SecurityBundle\Annotation\Acl;
+       use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+       use Symfony\Component\HttpFoundation\Request;
+
+       class TaskController extends Controller
+       {
+           /**
+            * @Acl(
+            *   id="app_task_view",
+            *   type="entity",
```

Comment:
I added a section that provides some more details: https://github.com/orocrm/documentation/pull/206/files#diff-f924240c2ac5eca3ac8df47bd7e79914R149